glean-core/android/src/main/java/mozilla/telemetry/glean/private/PingType.kt

```diff
@@ -17,7 +17,7 @@ import mozilla.telemetry.glean.rust.toByte
  * The Ping API only exposes the [send] method, which schedules a ping for sending.
  */
 class PingType(
-    name: String,
+    public val name: String,
```

Comment:
Maybe we should make this `internal` so it isn't part of the public API?  (It's low risk even if it is, though, so I don't feel too strongly about it).

Reply:
Yes, I think this is a good idea.